src/shiny/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '2.4'
+services:
+  app:
+    container_name: "application-server"
+    build:
+      context: .
+      dockerfile: Dockerfile
+    args:
+      - VARIANT=4
```

Comment:
Can you add a comment to the docker compose YAML file, or is there some place to document this configuration. How would someone maintaining this know what VARIANT 4 is and why it is set here?

Reply:
added a comment.

i wanted to make image tag as a templatable variable so user can pick something other than the default. same for other containers. but publishing a devcontainer template to ghcr.io requires a launch and I haven't gotten around to do it.